documentation/install-guide.html

```diff
@@ -134,6 +134,55 @@ <h2><a name="config"></a>Setup the Server</h2>
 Windows Server administrators should add http://127.0.0.1 address to Internet Explorer's Trusted Sites list, if
 Enhanced Security configuration is enabled in Internet Explorer. Otherwise they will get a blank screen.</p>
 
+<h3><a name="autosetup"></a>Autosetup</h3>
+<p>As an alternative to the web-based "wizard" driven setup described above, Openfire can be configured by provisioning it
+with a specially crafted <tt>openfire.xml</tt> file.</p>
+<p>If <tt>&lt;setup&gt;true&lt;>/setup&gt;</tt> does not exist in <tt>openfire.xml</tt> (i.e. setup is not complete, and a block
+does exist called <tt>&lt;autosetup&gt;</tt> with a <tt>&lt;run&gt;true&lt;/run&gt;</tt> property inside it, that looks like this:</p>
+<tt><pre>
+    &lt;autosetup&gt;
+        &lt;run&gt;true&lt;/run&gt;
+        &lt;locale&gt;en&lt;/locale&gt;
+        &lt;xmpp&gt;
+            &lt;domain&gt;localhost&lt;/domain&gt;
+            &lt;fqdn&gt;localhost&lt;/fqdn&gt;
+            &lt;auth&gt;
+                &lt;anonymous&lt;true&lt;/anonymous&lt;
```

Comment:
&lt;anonymous&lt;true&lt;/anonymous&lt; should be replaced with &lt;anonymous&gt;true&lt;/anonymous&gt;

Reply:
Good catch! I'll fix it.